packages/common/src/content/fetch.ts

```diff
@@ -249,13 +249,13 @@ export const fetchHubContent = async (
   requestOptions: IRequestOptions
 ): Promise<IHubEditableContent> => {
   // NOTE: b/c we have to support slugs we use fetchContent() to get the item
-  // by telling it to not fetch any enrichments
-  // which we then fetch as needed after we have the item
+  // by telling it to not fetch any enrichments which we then fetch as needed after we have the item
+  // EXCEPTION: We ask it to fetch the metadata enrichment so we don't have to parse xml on the client
   const options = {
     ...requestOptions,
-    enrichments: [],
+    enrichments: ["metadata"],
```

Comment:
I have a few questions here:
1) Do we only want formal metadata for public items?
2) Do we want to fetch formal metadata from our search index or from AGO directly (and have to parse XML ourselves)?
3) Should we rely on the metadata fetching logic of `fetchContent` or should we fetch it in `fetchHubContent`?

Reply:
> Do we want to fetch formal metadata from our search index or from AGO directly (and have to parse XML ourselves)?

I think we should get it from AGO directly, and parse it on the client. That's what we do in enterprise already, and  that means we don't have to worry about only fetching metadata if not on enterprise or for public items.

> Should we rely on the metadata fetching logic of fetchContent or should we fetch it in fetchHubContent?

We should fetch it in `fetchHubContent` using the `enrichments` system below.